book/Gremlin-Graph-Guide.adoc

```diff
@@ -16728,16 +16728,16 @@ produced by this query.
 
 [source,groovy]
 ----
-[e:4736, c:LHR]     [e:6410, c:NRT]

```

Comment:
Good catch - that reflects the latest data set not the version that was originally used.